app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
There's no race condition if, as @allait originally suggested, the audit event id is passed in as part of the payload rather than the URL. 

A small issue that would arise with something like `/services/<service_id>/updates/acknowledge-including-previous` (with the audit event id in either the payload or the URL) is that there's an extra database lookup required to validate that the `service_id` passed in matches the object referenced by the audit event, because I think the object id stored in the audit table isn't the (string) service ID but rather the numeric id from the services table that we never really expose anywhere else. It's not the end of the world, but a weirdness that would have to be worked with.

@galund I don't know whether the existing "ack single event" endpoint is going away right now or not, but either way can imagine we might want to support both single- and multi-event acknowledgements, and in that case it would be useful to be able to distinguish from the URL which is which without referring to the docs, so I think it needs something other than just `/acknowledge`.  (Not necessarily "including previous" if you have other suggestions... naming things is hard.)